src/components/HeaderForm.js

```diff
@@ -713,6 +718,11 @@ class HeaderForm extends Component {
     };
   };
 
+  /* Function for simplify */
```

Comment:
This comment doesn't do very much to explain the function. It should say what the function needs to do. Or you could say this is the setter function for the simplify two-way binding to the button and let that imply what it is supposed to do.